openedx_caliper_tracking/kafka_settings.py

```diff
@@ -0,0 +1,30 @@
+"""
+Contains class for kafka settings.
+"""
+from django.conf import settings
+
+
+class KafkaSettings:
+    """
+    Contains methods to get kafka settings.
+    """
+
+    @staticmethod
+    def get_maximum_retries():
+        if hasattr(settings, 'CALIPER_KAFKA_SETTINGS'):
+            return settings.CALIPER_KAFKA_SETTINGS.get('MAXIMUM_RETRIES')
+
+    @staticmethod
+    def get_error_report_email():
+        if hasattr(settings, 'CALIPER_KAFKA_SETTINGS'):
+            return settings.CALIPER_KAFKA_SETTINGS.get('ERROR_REPORT_EMAIL')
+
+    @staticmethod
+    def get_end_point():
+        if hasattr(settings, 'CALIPER_KAFKA_SETTINGS'):
+            return settings.CALIPER_KAFKA_SETTINGS.get('END_POINT')
+
+    @staticmethod
+    def get_topic_name():
+        if hasattr(settings, 'CALIPER_KAFKA_SETTINGS'):
+            return settings.CALIPER_KAFKA_SETTINGS.get('TOPIC_NAME')
```

Comment:
there is a lot of repetitive code in this file. Can we refactor this? You can use `defaultdict` and populate it if values exist in settings.
```
KafkaSettings = defaultdict(str)
if hasattr(settings, 'CALIPER_KAFKA_SETTINGS'):
    kafka_settings['END_POINT'] = settings.CALIPER_KAFKA_SETTINGS.get('MAXIMUM_RETRIES')
    kafka_settings['END_POINT'] = settings.CALIPER_KAFKA_SETTINGS.get('END_POINT')
....
```

Even you can move the above lines of code in the same file, where we are using these values.

Reply:
@tasawernawaz 
Nope. We couldn't, I tried this approach but I couldn't get the required settings while running my tests so the whole tests failed. I came up with the idea of using `static_methods` after trying many others. Before that, sometimes it fails on local but sometimes it shows errors just on `dev`.



# Tests Execution when `default dict` was used

![with_default_dict](https://user-images.githubusercontent.com/42294172/67739899-cd888e00-fa35-11e9-8c9f-bdfdb04821cc.gif)


# Tests Execution when `staticmethods` were used

![with_static_methods](https://user-images.githubusercontent.com/42294172/67739985-150f1a00-fa36-11e9-90e5-6fb5199f36a8.gif)

Reply:
@tasawernawaz @danialmalik 

`KafkaSettings` class has been removed
And tests are also working fine.